app/assets/javascripts/views/time_table_by_day.js

```diff
@@ -0,0 +1,169 @@
+App.Views.TimeTableByDay = Backbone.View.extend({
+  el: '.time-table',
+  template: JST['templates/time_table_by_day'],
+
+  initialize: function(options) {
+    var that = this;
```

Comment:
During my work I saw only `var self = this`. Where did you find `that`? It seems a little bit confusing.